packages/cacao/src/siwx/siwe.ts

```diff
@@ -13,22 +14,31 @@ export class SiweMessage extends SiwxMessage {
     return super.toMessage('Ethereum')
   }
 
+  toMessageEip55(): string {
+    const tmpAddress = this.address
+    this.address = checksumAddress(this.address as `0x${string}`)
+    const msg = super.toMessage('Ethereum')
+    this.address = tmpAddress
```

Comment:
Seems pretty hacky to have to change the address on the instance twice like this to make it work, could the address be provided as extra argument to the `toMessage()` method instead?